quickstart/test/quickstart_test.go

```diff
@@ -300,12 +300,15 @@ func TestQuickStartEnvFile(t *testing.T) {
 }
 
 /*
-This is a manually run test that will, with the default values, confirm the docker-compose ziti network is running as
-expected. The values can be edited to confirm other ziti networks but will require an http server on the back end.
+This is a manually run test that will, with the default values except the admin password, confirm the docker-compose
+ziti network is running as expected. The values can be edited to confirm other ziti networks but will require an http
+server on the back end.
 */
 func TestSimpleWebService(t *testing.T) {
 
 	// Wait for the controller to become available
+	zitiAdminUsername := "admin"
```

Comment:
@gberl002 can you just peek at this and make sure you're ok with this approach too, please